src/debugpy/adapter/clients.py

```diff
@@ -162,7 +162,7 @@ def initialize_request(self, request):
             "exceptionBreakpointFilters": [
                 {"filter": "raised", "label": "Raised Exceptions", "default": False},
                 {"filter": "uncaught", "label": "Uncaught Exceptions", "default": True},
-                # {"filter": "userUnhandled", "label": "User Uncaught Exceptions", "default": False},
+                {"filter": "userUnhandled", "label": "User Uncaught Exceptions", "default": False},
             ],
         }
```

Comment:
This needs a feature flag, so that the Python extension can conditionally enable it for an A/B test. To keep it simple, let's do it via an environment variable - something like `DEBUGPY_USER_UNHANDLED_EXCEPTION_FILTER` .

Reply:
I'll do it as a separate PR.